### DISCUSSION TITLE
What to use in v5 in place of point.Set method?

### DISCUSSION THREAD

**Discussion by BigBoulard** [jackc/pgx#1564] [Q&A] [answered]
Hi,

I'm migrating to `pgx v5`, and I can see that the `pgtype.Point` changed quite a bit.

v5: <https://github.com/jackc/pgx/blob/master/pgtype/point.go>
legacy: <https://github.com/jackc/pgtype/blob/master/point.go>

I can see that `Status` is no more there and that Valid made its entrance so I've changed my tests like below:

```go
// legacy
if p.Status != pgtype.Present { }
// v5
if !p.Valid { }
```

But what can I use to set a point from a String? I used to do this: 

```go
p.Set("(48.006110,0.199556)")
```

Thank you so much.

**Comment by jackc** [comment, marked as answer]
There isn't any specific parse string functionality, but I think you could use `Scan`.

Also, if this was immediately going to be used as a query argument you could skip the step and use the string directly.
